# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/vm/GlobalObject.h
# Commit: e590eb34efc5
# Full Hash: e590eb34efc52ab109f2737167cb2aa0975a1792
# Author: Iain Ireland <iireland@mozilla.com>
# Date: 2021-06-01 09:43:14
# Regressor Bug: 1554524
# File Overlap Count: 1
# Description:
#   Bug 1624792: Remove Debugger interface r=tcampbell
#   
#   This patch removes the changes from bug 1554524 part 4. (Parts 1 and 3 of bug 1554524 were bytecode emitter cleanups that don't need to be reverted; part 2 was abandoned.)
#   
#   Depends on D116308
# ==============================================================================

diff -r 135a0e6af641 -r e590eb34efc5 js/src/vm/GlobalObject.h
--- a/js/src/vm/GlobalObject.h	Tue Jun 01 01:31:21 2021 +0000
+++ b/js/src/vm/GlobalObject.h	Tue Jun 01 01:31:21 2021 +0000
@@ -122,7 +122,6 @@
     FOR_OF_PIC_CHAIN,
     WINDOW_PROXY,
     GLOBAL_THIS_RESOLVED,
-    INSTRUMENTATION,
     SOURCE_URLS,
     REALM_KEY_OBJECT,
     ARRAY_SHAPE,
@@ -884,15 +883,6 @@
     setReservedSlot(WINDOW_PROXY, ObjectValue(*windowProxy));
   }
 
-  JSObject* getInstrumentationHolder() const {
-    Value v = getReservedSlot(INSTRUMENTATION);
-    MOZ_ASSERT(v.isObject() || v.isUndefined());
-    return v.isObject() ? &v.toObject() : nullptr;
-  }
-  void setInstrumentationHolder(JSObject* instrumentation) {
-    setReservedSlot(INSTRUMENTATION, ObjectValue(*instrumentation));
-  }
-
   JSObject* getSourceURLsHolder() const {
     Value v = getReservedSlot(SOURCE_URLS);
     MOZ_ASSERT(v.isObject() || v.isUndefined());